### PR TITLE
Wip improvements

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
new readme and method name. Going to move this to object relations unit b/c song and artist are associated with one another. 

@jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,66 @@
...
+## Writing Data to Files in Ruby
+
+Ruby makes it easy to write data to a file. Ruby has a built-in [File class](http://ruby-doc.org/core-2.2.2/File.html). You can create new temporary files using: 
+
+`Tempfile.new`
+
+The `.new` method can take in a number of arguments. The first argument is an array. The first element of that array is the file name and the second argument of the array is the file extension/type. The second argument is the desired location of the file. Let's take a look: 
+
+```ruby
+my_file = Tempfile.new(["my_new_file", ".txt"], "desktop/my_aswesome_folder")
```
Comment (jmburges) [inline README.md]: awesome spelling

File: README.md [hunk 2]
```diff
@@ -1,22 +1,66 @@
...
+1. Understand why we as programmers need to manipulate and format data.
+2. Practice formatting data. 
+3. Introduce the topic of object relations. 
+4. Learn about writing data to files using Ruby. 
+
+## Data Normalization
+
+As programmers, it's our job to deliver information to people. We are being entrusted with an enormous responsibility––to handle people's data (whether it's videos, music, insurance information, pictures of cats, etc.) and deliver it to them in a clean, sensical manner. One very common action that you'll likely facilitate as a programmer is that of downloading files, such as audio files. Normal people like their files titled in normal ways. `artistSongxy1288304` doesn't mean anything to the muggles (non-progammers) of the world. In this lab, we'll be taking file names, normalizing them and writing their contents to a file in a `tmp` folder. 
+
+## Writing Data to Files in Ruby
```
Comment (jmburges) [inline README.md]: I would separate out the writing data part, from the lab

File: README.md [hunk 3]
```diff
@@ -1,22 +1,66 @@
...
-* Implement a song class and an artist class. Look at the specs or run your testing suite to see what the classes should look like.
-* Write a `serialize` method in the song class that:
-  * Replaces spaces in the song title with underscores and makes every character lower case.
-  * Uses this serialized version of the title as the name of a new [temporary](http://www.ruby-doc.org/stdlib-1.9.3/libdoc/tempfile/rdoc/Tempfile.html) text (`.txt`) file and saves it in the `tmp/` folder. 
-  * Writes the name of the artist then the name of the song separated by spaces and a dash (`-`) in the file.
+This is a test-driven lab. Use the test output and the guidelines below.
+
+### Associating Songs and Artists
+
+* Make two classes, Song and Artist. Songs have an `artist` attr_accessor (i.e. setter and getter) method. An individual artist's `song` attribute can be set equal to an instance of the Artist class.
```
Comment (jmburges) [inline README.md]: Song and Artist should be in backticks

File: README.md [hunk 4]
```diff
@@ -1,22 +1,66 @@
...
-  * Writes the name of the artist then the name of the song separated by spaces and a dash (`-`) in the file.
+This is a test-driven lab. Use the test output and the guidelines below.
+
+### Associating Songs and Artists
+
+* Make two classes, Song and Artist. Songs have an `artist` attr_accessor (i.e. setter and getter) method. An individual artist's `song` attribute can be set equal to an instance of the Artist class. 
+
+### Normalizing Data and Writing to Files
+
+The Song class should have a method `#slugify` that sanitizes the file name and saves a new file to the `tmp` folder.
```
Comment (jmburges) [inline README.md]: why are we making a method called `#slugify`. Like what is a slug?

Comment (SophieDeBenedetto) [inline README.md]: will add discussion of that idea/why we would do that here.

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md]: will make short readme to come before this lab that covers writing to files.

File: README.md (hunk 1 above)
Comment (SophieDeBenedetto) [inline README.md]: don't you mean asweome spelling?

Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: README.md (hunk 3 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: README.md (hunk 4 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

**Comment by jmburges** [comment]
:shipit:

**Pull request merged**